podtato-head-monolith/docker/DockerfileV0.1.2

```diff
@@ -0,0 +1,34 @@
+FROM library/golang:latest AS builder
```

Comment:
We could combine these 3 Dockerfiles into one and use an `arg` to set the building version.

```
docker build --build-arg VERSION=0.1.2 .
```

Only an idea to reduce code duplication.

Reply:
Makes sense to me. Again, perhaps we can merge in as things were previously and defer enhancements to follow-on PRs.